src/courseware/course/sequence/Unit/ContentIFrame.jsx

```diff
@@ -89,28 +89,16 @@ const ContentIFrame = ({
           <iframe title={title} {...contentIFrameProps} data-testid={testIDs.contentIFrame} />
         </div>
       )}
-      {modalOptions.isOpen && (modalOptions.isFullscreen
-        ? (
-          <ModalDialog
-            dialogClassName="modal-lti"
-            onClose={handleModalClose}
-            size="fullscreen"
-            isOpen
-            hasCloseButton={false}
-          >
-            <ModalDialog.Body className={modalOptions.modalBodyClassName}>
-              {modalContent}
-            </ModalDialog.Body>
-          </ModalDialog>
-
-        ) : (
-          <Modal
-            body={modalContent}
-            dialogClassName="modal-lti"
-            onClose={handleModalClose}
-            open
-          />
-        )
+      {modalOptions.isOpen && (
```

Comment:
[nit]: Should we still show `ModalDialog` with `size="fullscreen"` if modal options exist, and if not, instead of `Modal` (Deprecated), show `ModalDialog` with default `size` value?

Reply:
Added size condition `size={modalOptions.isFullscreen ? 'fullscreen' : 'md'}`